### DISCUSSION TITLE
AX53U

### DISCUSSION THREAD

**Discussion by CHazz** [SWRT-dev/swrt-gpl#33] [General]
hi, I'm looking for firmware for Asus AX53U, if I compile it, will it support AImesh? in the download section i found only version B without AImesh, so how is it? thank you very much

**Comment by paldier** [comment]
unsupport aimesh

**Reply by CHazz** [reply]
> unsupport aimesh

ASUS RT-AX53U Firmware version 3.0.0.4.386.67508
Verze 3.0.0.4.386.67508
37.77 MB
2022/05/30
1. Supported AiMesh function.

**Comment by CHazz** [comment]
the router supports AIMESH, of course, but you're probably using old and outdated firmware that didn't support it yet, too bad... thank you for the answer 😔
